apps/docs/content/guides/auth/auth-hooks.mdx

```diff
@@ -313,23 +313,33 @@ Hooks return status codes based on the nature of the response. These status code
 | HTTP Status Code | Description                                                   | Example Usage                                  |
 | ---------------- | ------------------------------------------------------------- | ---------------------------------------------- |
 | 200, 202, 204    | Valid response, proceed                                       | Successful processing of the request           |
-| 429, 503         | Retry-able errors with Retry-after header supported           | Temporary server overload or maintenance       |
 | 403, 400         | Treated as Internal Server Errors and return a 500 Error Code | Malformed requests or insufficient permissions |
+| 429, 503         | Retry-able errors                                             | Temporary server overload or maintenance       |
 
-Errors are responses which contain status codes 400 and above. On a retry-able error, such as an error with a `429` or `503` status code, HTTP Hooks will attempt up to three retries with a back-off of two seconds.
+<Admonition type="note">
+
+`204` See the hook specific documentation for status code handling. Some hooks do not support `204` responses because they read the request body.
```

Comment:
```suggestion
`204` Read the hook specific documentation for status code handling. Some hooks do not support `204` responses because they read the request body.
```

Reply:
Could we add a link to that documentation?